student-bios/emran-ai.md

```diff
@@ -1,11 +1,16 @@
 # Introduction
 
-Hi, everyone! My name is Emran and I'm a student. I'm excited to be part of this class and to learn from all of you.
+Hi, everyone! My name is Emran and I'm a student. I'm excited to be part of this
```

Comment:
I am a bit confused about this commit, the message says 
> add item to .c spell.json
but you just changed the link to your image (line 16)